src/Gemini/AppBootstrapper.cs

```diff
@@ -189,10 +189,10 @@ protected override IEnumerable<object> GetAllInstances(Type serviceType)
         protected override void BuildUp(object instance)
             => Container.SatisfyImportsOnce(instance);
 
-        protected override void OnStartup(object sender, StartupEventArgs e)
+        protected override async void OnStartup(object sender, StartupEventArgs e)
```

Comment:
Since this is `async void`, the calling method won't wait for the result of `DisplayRootViewForAsync` if that method is asynchronous - is that a problem?

Reply:
Good catch! That's what I get for coding after midnight :). This problem was also brought up in a recent CM Issue: https://github.com/Caliburn-Micro/Caliburn.Micro/issues/819. I'll resolve the latest conflicts and do a fix here.

Also noticing this comment by the current CM maintainer, which is probably a relevant topic for future Gemini updates:
```
I do need to drop support for .net 5 and .net core 3.1 will be out of support in December 2022

Caliburn Micro 5 will include support WinUI, Maui, and AvaloniaUI
```

Reply:
@tgjones, sorry for leaving this comment, I've navigated to this repo from the Caliburn Micro repo.

Making `OnStartup` asynchronous but non-awaited is not a problem, I think, due to the fact that `OnStartup` subscribes to the current `Application` instance, event `Startup`. When we handle this event we never rely on a code running afterward based on my experience in developing WPF applications.